lib/Notes/NoteViewPage/components/NoteView/tests/note-view-test.js

```diff
@@ -160,8 +167,6 @@ describe('NoteView', () => {
       );
     });
 
-    it('should screen suspicious content for display', () => {
-      expect(noteView.noteContentText).to.equal(cleanContent);
-    });
+    it('should screen suspicious content for display', () => NoteContent('Details').has({ markup: cleanContent }));
```

Comment:
Are we really screening "suspicious" content here or are we just sucking all potentially-malicious text through the DOMPurify filter? I think it's the latter, so maybe "should screen values for malicious content" or "should screen content for XSS" or such like?

Reply:
It's a test. It's all suspicious.